doc/23_Installation_and_Upgrade/07_Updating_Pimcore/12_V10_to_V11.md

```diff
@@ -2,6 +2,15 @@
 
 ## IMPORTANT CHANGES TO DO PRIOR THE UPDATE!
 
+### Migrations
+Make sure that migrations are executed.
+Pimcore does only run Pimcore core migrations after `composer` update automatically. If you'd like to run all available migrations including bundles and your app-specific migrations, just include the following part in your `composer.json`.
+```json
+"post-update-cmd": [
+    "./bin/console doctrine:migrations:migrate"
```

Comment:
This should be enough, right? (Because of https://getcomposer.org/doc/articles/scripts.md#:~:text=Note%3A%20Before,found%20and%20executed)

```suggestion
    "console doctrine:migrations:migrate"
```

Reply:
I tried it, but I got the following error:
```bash
sh: 1: console: not found
Script console doctrine:migrations:migrate handling the post-update-cmd event returned with error code 127
```
I would keep it as is.